mintapi/api.py

```diff
@@ -568,7 +572,7 @@ def get_transactions_json(self, include_investment=False,
             # transactions as well.  Otherwise they are skipped by
             # default.
             if id > 0 or include_investment:
-                params['id'] = id
+                params['accountId'] = id
```

Comment:
Looks good overall. What is this change for?

Reply:
This was the typo in commit e069366 that broke retrieving transactions for a specified account. This fixes it.